compiler/rustc_mir_transform/src/lower_slice_len.rs

```diff
@@ -10,6 +10,10 @@ use rustc_middle::ty::{self, TyCtxt};
 pub struct LowerSliceLenCalls;
 
 impl<'tcx> MirPass<'tcx> for LowerSliceLenCalls {
+    fn is_enabled(&self, sess: &rustc_session::Session) -> bool {
+        sess.opts.mir_opt_level() > 0
+    }
```

Comment:
In `master`, we check that `mir_opt_level > 0` in `mir_optimized` before running this pass (and many others). That's why it has no analogous check in `run_pass`.